src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/MethodWriter.java

```diff
@@ -105,13 +105,13 @@ protected void buildMethodDoc(Content detailsList) {
             for (Element method : methods) {
                 currentMethod = (ExecutableElement)method;
```

Comment:
As a separate RFE, _it would be nice_ if `getVisibleMembers` took an optional `Class<T>` argument that is used to cast the result of the method to `List<T extends Element>` instead of just `Element`.  This would enable to eliminate downstream casts, like here on line 106, to get the subtype.

See a similar technique in `Utils` for accessing sublists of a list of `DocTree` nodes,